cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -2712,6 +2712,15 @@
   </property>
 
   <property>
+    <name>security.authorization.cache.max.entries</name>
+    <value>10000</value>
```

Comment:
This is based on a single principal owning 2500 entities for user containers. Unfortunately, the cache does not have a size based limit.